README.md

```diff
@@ -2,6 +2,8 @@
 
 A web application that shows various lockdown states for countries across the world helping people to find out if they can travel and at what restrictions.
 
+Required Components: Node Version 10.0, preact v10
```

Comment:
its not really worth mentioning Preact here, since npm install will just install it for you. Npm requires node 10^, so we can mention it here
```suggestion
Required Components: Node Version 10.
```

Reply:
@thepassle: ok noted. changing to Node Version 10.